third_party/move/move-vm/runtime/src/interpreter.rs

```diff
@@ -202,8 +241,17 @@ impl InterpreterImpl {
             self.active_modules.insert(module_id.clone());
         }
 
+        let frame_cache = std::rc::Rc::new(std::cell::RefCell::new(Default::default()));
```

Comment:
`Default::default()` doesn't contribute to readability in a context like this, perhaps use the type name.